CMakeLists.txt

```diff
@@ -69,3 +69,5 @@ add_subdirectory(libtempered)
 add_subdirectory(libtempered-util)
 add_subdirectory(utils)
 add_subdirectory(examples)
+
+find_package(rock)
```

Comment:
In control/visp, you actually use the rock macros to generate a pkg-config file ... it seems. I'm not even sure that it does if you don't use rock_executable.